packages/utils/src/sourceCredHelpers.ts

```diff
@@ -2,13 +2,19 @@ import { ethers } from 'ethers';
 import { SCIdentity, sourcecred as sc } from 'sourcecred';
 
 export const getLatestEthAddress = (identity: SCIdentity): string | null => {
-  const ethAddress = identity.aliases.find((alias) => {
+  const ethAddresses = identity.aliases.filter((alias) => {
     const parts = sc.core.graph.NodeAddress.toParts(alias.address);
-    return parts.indexOf('ethereum') > 0;
-  })?.description;
-
-  if (ethAddress && ethers.utils.isAddress(ethAddress)) {
-    return ethAddress.toLowerCase();
+    return (
+      parts.indexOf('ethereum') > 0 && ethers.utils.isAddress(alias.description)
+    );
+  });
+  if (ethAddresses.length > 0) {
+    // this assumes that the fetched data is in chronological order, which should
+    // be the case for the SC Ledger
+    const latestAddress = ethAddresses.pop();
+    if (latestAddress != null) {
+      return latestAddress?.description.toLowerCase();
+    }
```

Comment:
Since you're using the nullish coalescing operator, you can let `latestAddress` be `undefined`.

If, for whatever reason you wanted to always return `null` on a failure, you could do `return latestAddress?.description.toLowerCase() ?? null`.

Since the filter only passes aliases with valid ETH addresses as descriptions, if we make it past the empty array check, I don't see how the `pop` could be `undefined` anyhow.

```suggestion
    return latestAddress?.description.toLowerCase();
```

Reply:
All you'd have to do is hit the "Commit Suggestion" button. The `if` is unarguably superfluous.

As always, it's just a suggestion.